app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -63,6 +68,21 @@ def handle_success(submission)
     form_submission_attempt.vbms!
   end
 
+  def batch_process(pending_form_submissions)
+    total_handled = 0
+
+    pending_form_submissions.each_slice(max_batch_size) do |batch|
```

Comment:
😮 another rubyism I didnt know. I dig it.

Reply:
Yeah it's very handy for this sort of thing! @pennja found it!